src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -33,6 +33,17 @@ public override string StringCode
         set => throw new NotImplementedException();
     }
 
+    public override void RepositionToOrigin()
+    {
+        var centerOfMass = Organelles.CenterOfMass;
+
+        foreach (var organelle in Organelles)
+        {
+            // This calculation aligns the center of mass with the origin by moving every organelle of the microbe.
+            organelle.Position -= centerOfMass;
```

Comment:
I assume that even if this gets called multiple times it doesn't cause problems, right?

Reply:
no, because the second time this code runs the center of mass should be 0,0